EXILED/Exiled.API/Features/Waves/TimedWave.cs

```diff
@@ -21,40 +21,43 @@ namespace Exiled.API.Features.Waves
     /// </summary>
     public class TimedWave
```

Comment:
can we add the IWrapper ?

Reply:
IWrapper still requires MonoBehaviour since it hasn't been merged yet, it's in the SSSS pr